tooltester/src/main/java/io/dockstore/tooltester/helper/S3CacheHelper.java

```diff
@@ -4,24 +4,22 @@
  * @author gluu
  * @since 18/09/17
  */
-public class S3CacheHelper {
+public final class S3CacheHelper {
+
+    private S3CacheHelper() {
+        // hidden constructor
+    }
+
+    @SuppressWarnings("checkstyle:Indentation")
     public static String mapRepositoryToCache(String repository) {
-        switch(repository) {
-        case "github.com/briandoconnor/dockstore-workflow-md5sum":
-            return "ga4gh-dream/md5sum";
-        case "github.com/Barski-lab/ga4gh_challenge":
-            return "ga4gh-dream/biowardrobe";
-        case "github.com/NCI-GDC/gdc-dnaseq-cwl/GDC_DNASeq":
-            return "ga4gh-dream/gdc";
-        case "github.com/bcbio/bcbio_validation_workflows":
-            return "ga4gh-dream/bcbio";
-        case "github.com/ENCODE-DCC/pipeline-container/encode-mapping-wdl":
-        case "github.com/ENCODE-DCC/pipeline-container/encode-mapping-cwl":
-            return "ga4gh-dream/encode";
-        case "github.com/KnowEnG/cwl-gene-prioritization":
-            return "ga4gh-dream/knoweng";
-        default:
-            return "";
-        }
+        return switch (repository) {
+            case "github.com/briandoconnor/dockstore-workflow-md5sum" -> "ga4gh-dream/md5sum";
```

Comment:
Cool, didn't know you could use `->` with `switch`

Reply:
Yeah, the only tricky bit is it doesn't look like checkstyle understands it yet, hence the 
`@SuppressWarnings("checkstyle:Indentation")`